src/mds/MDBalancer.cc

```diff
@@ -493,8 +493,14 @@ void MDBalancer::queue_split(const CDir *dir, bool fast)
 
     // Pass on to MDCache: note that the split might still not
     // happen if the checks in MDCache::can_fragment fail.
+    auto mds_bal_split_bits = g_conf->mds_bal_split_bits;
```

Comment:
need to make sure "split_dir->get_frag().bits() + mds_bal_split_bits " <= 24

Reply:
if that's required, this isn't correct is it?
https://github.com/ceph/ceph/blob/master/src/common/options.cc#L6216-L6218